libs/agora/services/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "agora-services",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "projectType": "library",
+  "sourceRoot": "libs/agora/services/src",
+  "prefix": "agora",
+  "targets": {
+    "test": {
+      "executor": "@nx/jest:jest",
+      "outputs": ["{workspaceRoot}/coverage/libs/agora/services"],
+      "options": {
+        "jestConfig": "libs/agora/services/jest.config.ts"
```

Comment:
```suggestion
        "jestConfig": "{projectRoot}/jest.config.ts"
```

Reply:
This makes the file more portable.

Reply:
great suggestion!  I copied this from somewhere else so this pattern could be applied in more places in the monorepo